servers/Template.Mcp.Server/src/Template.Mcp.Server.csproj

```diff
@@ -5,18 +5,29 @@
     <AssemblyTitle>Template MCP Server</AssemblyTitle>
     <Description>Template MCP Server - Basic Model Context Protocol implementation</Description>
     <ReadmeUrl>https://github.com/Microsoft/mcp/blob/main/servers/Template.Mcp.Server#readme</ReadmeUrl>
+    <ReadmePath>$(RepoRoot)/servers/Template.Mcp.Server/README.md</ReadmePath>
+
+    <!-- npm settings -->
     <NpmPackageName>@azure/mcp-template</NpmPackageName>
     <NpmPackageKeywords>azure,mcp,model-context-protocol</NpmPackageKeywords>
-    <PackageId>Microsoft.Template.Mcp</PackageId>
-    <ToolCommandName>$(CliName)</ToolCommandName>
+
+    <!-- docker settings -->
+
+    <!-- dnx/nuget settings -->
+    <DnxPackageId>Microsoft.Template.Mcp</DnxPackageId>
+    <DnxToolCommandName>$(CliName)</DnxToolCommandName>
+    <DnxPackageTags>template,mcp</DnxPackageTags>
+
+    <!-- vsix settings -->
+    <VsixPackagePrefix></VsixPackagePrefix>
   </PropertyGroup>
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <AssemblyName>$(CliName)</AssemblyName>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <!-- Publishing settings -->
-    <IsAotCompatible>true</IsAotCompatible>
+    <IsAotCompatible>false</IsAotCompatible>
```

Comment:
Was this intentional? I mean, does the template already contain AOT-incompatible code?